packages/round-manager/src/context/round/UpdateRoundContext.tsx

```diff
@@ -157,13 +158,22 @@ const _updateRound = async ({
       transactionBuilder.add(UpdateAction.UPDATE_ROUND_FEE_PERCENTAGE, [arg]);
     }
 
+    /* Special case - if the round has already started, and we are editing times,
+     * we need to set newApplicationsStartTime and newRoundStartTime to something bigger than the block timestamp.
+     * This won't actually update the values, it's done just to pass the checks in the contract
+     * (and to confuse the developer). */
+    if (Date.now() > round.roundStartTime.getTime()) {
+      round.applicationsStartTime = subSeconds(round.applicationsEndTime, 1);
+      round.roundStartTime = subSeconds(round.applicationsEndTime, 1);
+    }
```

Comment:
let's explain why in the comment, it makes sense to remember that the contract call doesn't fail, 
it updates the end time only if the round hasn't started yet

Reply:
Could you be more specific here? Not sure what you want to change about the explanation.
Feel free to suggest the changes to be commited directly in the review here.

Reply:
TODO: link the contract checks here